File: trail1/2가 3번째로 등장하는 위치/where-2-appears-3rd.c
#include <stdio.h>
#include <stdlib.h>

int main() {

    int n;
    
    scanf("%d ", &n);

    int* arr = malloc(sizeof(int)*n);

    for(int i = 0; i < n; i++){
        scanf("%d ", &arr[i]);
    }

    int count = 0;
    for(int j = 0; j < n; j++){
        if(arr[j]==2){
            count++;
        }

        if(count == 3){
            printf("%d", j+1);
            break;
        }
    }

    free(arr);
    return 0;
}

File: trail1/2가 3번째로 등장하는 위치/where-2-appears-3rd.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
       
       int n = sc.nextInt();

       int[] arr = new int[n];
       
        for(int i = 0; i < n; i++){
            arr[i] = sc. nextInt();
        }
        
        int count = 0;
        for(int j = 0; j < n; j++){
            if(arr[j]==2){
                count++;
            }

            if(count == 3){
                System.out.printf("%d", j+1);
                break;
            }
        }
    }
}